packages/admin-ui/src/filters/filter-group.tsx

```diff
@@ -12,7 +12,16 @@ export function FilterGroup(props: FilterGroupProps) {
   const formatMessage = useMessageFormatter(messages.actions)
 
   return (
-    <Flex wrap="wrap" role="group" {...rest}>
+    <Flex
+      wrap="wrap"
+      role="group"
+      csx={{
+        '> .__admin-ui-filter-disclosure:not(:first-child)': {
+          marginLeft: '$m',
+        },
+      }}
+      {...rest}
+    >
       {children}
```

Comment:
Using the inline component won't work?
```suggestion
    <Inline hspace="$m">
      {children}
    </Inline>
```

Reply:
That adds space to the popovers, which shouldn't happen